src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -1091,4 +794,85 @@ input TimestampConfiguration {
   createdAt: String
   updatedAt: String
 }
-```
\ No newline at end of file
+```
+
+### Relational directives
+The relational directives are `@hasOne`, `@hasMany`, `@belongsTo` and `@manyToMany`.
+<BlockSwitcher>
+<Block name="@hasOne">
+
+The `@hasOne` will generate:
+
+- Foreign key fields in parent type that refer the the primary key and sort key fields of the child model.
+- Foreign key fields in parent input object of `create` and `update` mutations.
+
+<Callout>
+The foreign key fields can be customized by adding existing fields in `fields` attribute.
+</Callout>
+
+**Type definition of the `@hasOne` directive**
+
+```graphql
+directive @hasOne(fields: [String!]) on FIELD_DEFINITION
+```
+</Block>
+<Block name="@hasMany">
+
+The `@hasMany` will generate:
+
+- Foreign key fields in child type that refer the the primary key and sort key fields of the parent model.
+- Foreign key fields in child input object of `create` and `update` mutations.
+- A global secondary index(GSI) in the child type Amazon DynamoDB table.
```

Comment:
```suggestion
- A global secondary index (GSI) in the child type AWS DynamoDB table.
```

Reply:
I check the DynamoDB console. It uses `Amazon` instead of `AWS`

Reply:
Oh okay, sounds good to me!